examples/azure-rules.yaml

```diff
@@ -756,63 +756,25 @@ groups:
     description: "Check that <sidadm> sudo user is set up in /etc/sudoers"
     checks:
       - id: 1.6.1
-        description: "Basic sudoers entry to allow <sidadm> to use the srHook."
-        audit: 'cat /etc/sudoers | grep -E ".+ +ALL=\(ALL\) +NOPASSWD: +/usr/sbin/crm_attribute +-n +hana_.+_site_srHook_\*" | sed "s/.*/true/"'
+        description: "Allowing <sidadm> to access the Cluster."
+        audit: |
+          cat /etc/sudoers | grep -E ".+ +ALL=\(ALL\) +NOPASSWD: +/usr/sbin/crm_attribute +-n +hana_[a-z][a-z0-9]{2}_site_srHook_\*" | sed "s/.*/true1/"
+          cat /etc/sudoers | grep -E "^Cmnd_Alias +SOK_SITEA += +/usr/sbin/crm_attribute" | grep -E "\-n +hana_[a-z][a-z0-9]{2}_site_srHook_.+" | grep -E "\-v +SOK" | grep -E "\-t +crm_config" | grep -E "\-s +SAPHanaSR" | sed "s/.*/true2/"
+          cat /etc/sudoers | grep -E "^Cmnd_Alias +SFAIL_SITEA += +/usr/sbin/crm_attribute" | grep -E "\-n +hana_[a-z][a-z0-9]{2}_site_srHook_.+" | grep -E "\-v +SFAIL" | grep -E "\-t +crm_config" | grep -E "\-s +SAPHanaSR" | sed "s/.*/true3/"
+          cat /etc/sudoers | grep -E "^Cmnd_Alias +SOK_SITEB += +/usr/sbin/crm_attribute" | grep -E "\-n +hana_[a-z][a-z0-9]{2}_site_srHook_.+" | grep -E "\-v +SOK" | grep -E "\-t +crm_config" | grep -E "\-s +SAPHanaSR" | sed "s/.*/true4/"
+          cat /etc/sudoers | grep -E "^Cmnd_Alias +SFAIL_SITEB += +/usr/sbin/crm_attribute" | grep -E "\-n +hana_[a-z][a-z0-9]{2}_site_srHook_.+" | grep -E "\-v +SFAIL" | grep -E "\-t +crm_config" | grep -E "\-s +SAPHanaSR" | sed "s/.*/true5/"
+          cat /etc/sudoers | grep -E "[a-z][a-z0-9]{2}adm +ALL=\(ALL\) +NOPASSWD:" | sed "s/.*: //" | grep SOK_SITEA | grep SFAIL_SITEA | grep SOK_SITEB | grep SFAIL_SITEB | sed "s/.*/true6/"
         tests:
           test_items:
-          - flag: "true"
+          - flag: "true1"
```

Comment:
I think this is missing a "set: true" to ensure that missing the output will not pass the check. 

see https://github.com/aquasecurity/bench-common/blob/main/docs/README.md#check

Reply:
@dirkmueller  What they mean in https://github.com/aquasecurity/bench-common/blob/main/docs/README.md#check is that they check a specific flag that it equals (eq) to some value like `--flag=value`. In this code I check that those flags exist at all and they exist alltogether. My check is somewhat like here https://github.com/trento-project/trento/blob/main/examples/azure-rules.yaml#L446-L453
I haven't specified that `bin_op: and` because it's the default.

Reply:
ah, I missed the `bin_op: and` being default, thank you for the explanation!